tools/tesseratos/include/tesseratos/metrics_panel/plugin.hpp

```diff
@@ -0,0 +1,18 @@
+/// @dir
+/// @brief @ref tesseratos-metrics-panel-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup tesseratos-metrics-panel-plugin
+
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+
+namespace tesseratos
+{
```

Comment:
You forgot to include this docs block, which creates a page for this plugin in the docs:

```suggestion
{
    /// @defgroup tesseratos-metrics-panel-plugin Metrics
    /// @ingroup tesseratos
    /// @brief Shows some useful performance metrics through a ImGui window.
    ///
    /// ## Dependencies
    /// - @ref imgui-plugin

```

Reply:
eu tinha esses comentários mas disseram para tirar 😭